firebase/functions/index.js

```diff
@@ -0,0 +1,50 @@
+// @license
```

Comment:
It looks pretty straightforward, I see the notes on get/set the config values in README above. Two thoughts:
- for the production instance are they also avail in the web UI console somehow?
- we should have the keys written down and/or checked in somewhere maybe in a private GitHub or otherwise, and/or, info on the Google API console login and project name, etc. to retrieve them.

Reply:
Thanks! Yeah, this is pretty simple.

The keys aren't available in the web UI anywhere (at least not that I've been able to find).

I'll either use our current arcs firebase, or a new instance that we all have access to. It'll be easiest to just view the environment variable directly from there, at least until we lock access down. I'll expand the instructions in the README though to make the process clearer.